projects/vdk-core/src/vdk/api/job_input.py

```diff
@@ -151,8 +151,10 @@ def send_object_for_ingestion(
         The data is then processed and sent asynchronously by multiple background threads in parallel
          (the level of parallelism could be controlled by ingester_number_of_worker_threads configuration).
 
+        Alternatively, the same method can be used to send a pandas.DataFrame for ingestion, asynchronously.
+
         Arguments:
-            payload: dict
+            payload: Union[dict, pandas.DataFrame]
```

Comment:
put pandas.DataFrame in "" "pandas.DataFrame" 

Otherwise some IDEs may decide to add the import